.github/.wordlist.txt

```diff
@@ -312,6 +312,7 @@ customizations
 cvfJ
 cxx
 CY
+cypressprogrammer
```

Comment:
cypresprogrammer is not a word.

Use URL tags or backticks to not need to add these as words

Reply:
we have lots of words (even listed above that aren't words)

Reply:
@andy31415 : Remove the words by adding URL tags